README.md

```diff
@@ -68,22 +68,29 @@ $ export KUBECONFIG=/path/to/admin.kubeconfig
   ```
   $ make run-local
   ```
-4. Open a new terminal window and create an Infinispan cluster with three nodes:
+4. Optionally install the Infinispan operator on OKD from the bundle file.
+  * Create a new project or use an existing one:
+  ```
+  $ oc new-project infinispan-operator
+  ```
+  * Run this command to install the operator:
+  ```
+  $ oc apply -n infinispan-operator -f https://raw.githubusercontent.com/infinispan/infinispan-operator/master/deploy/operator-install.yaml
```

Comment:
I would use the same format for all the `oc apply` commands: below we're using local file and no project specified